front/app/containers/Admin/reporting/components/ReportBuilder/Toolbox/index.tsx

```diff
@@ -65,10 +74,36 @@ const SectionTitle = ({ children }) => (
 const ReportBuilderToolbox = ({ reportId }: ReportBuilderToolboxProps) => {
   const { formatMessage } = useIntl();
   const formatMessageWithLocale = useFormatMessageWithLocale();
-  const { projectId, phaseId } = useReportContext();
+  const { projectId } = useReportContext();
   const appConfigurationLocales = useAppConfigurationLocales();
+  const { data: authUser } = useAuthUser();
+  const isModerator = !!authUser && !isAdmin(authUser);
```

Comment:
Should we use `isModerator` from `front/app/utils/permissions/roles.ts` here? Not a big deal as anyway it works correctly in this context. Just in case something changes or this code is copied to the front office.